lib/pbench/cli/server/shell.py

```diff
@@ -92,12 +61,14 @@ def run_gunicorn(server_config: PbenchServerConfig, logger: Logger) -> int:
         es_wait_timeout = int(server_config.get("Indexing", "wait_timeout"))
         workers = str(server_config.get("pbench-server", "workers"))
         worker_timeout = str(server_config.get("pbench-server", "worker_timeout"))
-        crontab_dir = server_config.get("pbench-server", "crontab-dir")
         server_config.get("flask-app", "secret-key")
     except (NoOptionError, NoSectionError) as exc:
         logger.error("Error fetching required configuration: {}", exc)
+        notifier.notify("STOPPING=1")
+        notifier.notify("STATUS=Unable to configure gunicorn")
```

Comment:
It would be nice if, barring reasons to the contrary, we were consistent about the order in which we post "notifications".  For instance, at lines 80-82 and 85-87 we notifiy `systemd` first and then log the error; here, we're doing it in the opposite order.

Ditto lines 184-185 and 187-189.

(I really like the notion of notifying `systemd`, though! 🙂)

Reply:
I was actually *hoping* that `systemd` would journal the status messages, and I could remove the direct journaling, so I didn't worry about the ordering. It doesn't, but I didn't go back to double-check. I don't see that consistency is any sort of meaningful difference, but in terms of just reading the code it might make some sense.

Reply:
It helps when reading the code, and it helps to ensure that we always (make sure) that we do them both...and, if we can maintain the consistency, then we can draw the inferences based on seeing one being sure that the other either happened or would have happened next (depending on which one we're looking at...).